sources/models/src/lib.rs

```diff
@@ -293,6 +294,25 @@ struct KernelSettings {
     sysctl: HashMap<SysctlKey, String>,
 }
 
+// Kernel boot settings
+#[model]
+struct BootSettings {
+    #[serde(
+        alias = "kernel",
+        rename(serialize = "kernel"),
+        default,
+        skip_serializing_if = "Option::is_none"
```

Comment:
The struct member's value isn't an `Option` so I think this may be a typo?  Should it be `HashMap::is_empty`, or excluded all together?

Reply:
With the `#[model]` attribute, all struct member fields are wrapped in `Option` by default. See https://github.com/bottlerocket-os/bottlerocket/blob/42bffd64bf9733394a317f961e6ff01852efe184/sources/models/model-derive/src/lib.rs#L152-L163